modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -764,7 +769,7 @@ public void preprocessOpenAPI(OpenAPI openAPI) {
                 Schema s = e.getValue();
                 String nOneOf = toModelName(n + "OneOf");
                 if (ModelUtils.isComposedSchema(s)) {
-                    if (e.getKey().contains("/")) {
+                    if (e.getKey().contains("/") || (useOneOfInterfaces && LIBRARY_ONEOF_IMPL.equals(templateDir))) {
```

Comment:
The code is specific to the SpringCodegen class and must be written in it???

Reply:
Yes, the code will be executed only for java spring library, as the specification of this issue require.

Reply:
If so, move the code in SpringCodegen.java and all if statments with  (.... templateDir.equals(LIBRARY_ONEOF_IMPL....) should be in SpringCodegen.java

Reply:
To do it so, i must duplicate almost the whole preprocessOpenAPI method in SpringCodegen, only because of this condition. I wanted to avoid it

Reply:
@Scorpioncik should i do so?

Reply:
You don't need to duplicate the whole preprocessOpenAPI, you can extract in a method the piece of code and override it in SpringCodegen.

example:
if (e.getKey().contains("/")) {      ====> if (method1()) {